core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -204,13 +228,16 @@ private[spark] class SparkClassNotFoundException(
 private[spark] class SparkConcurrentModificationException(
     errorClass: String,
     errorSubClass: Option[String] = None,
-    messageParameters: Array[String],
+    messageParameters: Map[String, String],
```

Comment:
The exception is used only once to raise the error class `CONCURRENT_QUERY` but the last one is not tested at all. I open the JIRA https://issues.apache.org/jira/browse/SPARK-40408 to add a test for it.